### PR TITLE
User Can See Branding and Navigation

### PR DESCRIPTION
**Pull request by youngmlee**
Adds branding and navigation to index.html.
screen shot 2017-09-01 at 2 27 54 pm 2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8" />
+    <title>Sway Finance</title>
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <img class="branding" src="images/user-can-see-branding-and-navigation.png" alt="Picture of logo and navigation options on top of webpage" />
```
Comment (victormejia) [inline index.html]: you can't just put the picture here. You have to re-create it using HTML and CSS.

**Review by victormejia — CHANGES_REQUESTED** [review]

**Comment by youngmlee** [comment]
Right sorry haha

**Comment by youngmlee** [comment]
Screenshot of progress:
screen shot 2017-09-01 at 10 31 41 pm

**Comment by victormejia** [comment]
Please fix the fonts on the navigation buttons in your next pull request.

**Pull request merged**